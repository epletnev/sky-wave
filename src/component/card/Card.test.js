import React from "react";
import MenuCard from "./Card";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";

test("should render MenuCard component", () => {
  const tree = renderer
    .create(
      <Router>
        <MenuCard
          item={{
            id: 1,
            name: "Snack Set",
            description:
              "Butter, cream, honey, salty cottage cheese, white cheese.",
            price: 8.2,
            image: {
              small: "assets/images/foods/breakfast/1/small.jpg",
            },
            availibilityCount: 5,
            weight: 160,
            isVegetarian: false,
            categoryId: "Breakfast",
          }}
        />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot;
});
