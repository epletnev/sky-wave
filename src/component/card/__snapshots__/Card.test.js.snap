// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expext to render Card component 1`] = `<MenuCard />`;
